cluster/libvirt-coreos/user_data_master.yml

```diff
@@ -16,7 +16,7 @@ coreos:
         ExecStart=/opt/kubernetes/bin/kube-apiserver \
         --address=0.0.0.0 \
         --port=8080 \
-        --etcd_servers=${etcd_servers} \
+        --etcd_servers=http://127.0.0.1:4001 \
```

Comment:
I think we should lock down etcd on localhost and have it be a single instance only.

In the etcd cluster scenario, etcd is obviously bound on host ips.
This leaves etcd exposed since it is unsecured.

Hence, For the libvrt/vireos case too , I think we should have it be a single etcd instance.

Once we have our etcd cluster story with proper access controls, we can enable it for all providers.



Sent from my iPhone

> On Apr 3, 2015, at 2:18 PM, Zach Loafman <notifications@github.com> wrote:
> 
> In cluster/libvirt-coreos/user_data_master.yml:
> 
> > @@ -16,7 +16,7 @@ coreos:
> >          ExecStart=/opt/kubernetes/bin/kube-apiserver \
> >          --address=0.0.0.0 \
> >          --port=8080 \
> > -        --etcd_servers=${etcd_servers} \
> > +        --etcd_servers=http://127.0.0.1:4001 \
> The Salt ones I'm comfortable with. Do you know if libvirt-coreos is launching the etcd pod yet? I suppose this shouldn't matter anyways, though, since it's running regardless, but in true Core style, it looks like libvirt-coreos is setting up an etcd cluster, too? https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/libvirt-coreos/util.sh#L194
> 
> —
> Reply to this email directly or view it on GitHub.
>